### ISSUE TITLE
Would it be possible to add an "any" sensor?

### ISSUE THREAD

**Issue by jacoscar** [e-mcgee/paradox_platform#30]
Well done first of all: this is the only paradox plugin that I managed to make work (plus it doesn't need mtqq and a separate python script).

I have created an automation in homekit so that when a window connected to the paradox alarm opens, then the Tado heating system turns off and turn on again when the window is closed.
I have to do this for each window and it's not 100% fool proof because HomeKit doesn't allow to create automations based on multiple sensors, but only single automations based on single sensors.

So if I open two windows but then close only one of them, the heating turns on again, while I would like to turn it on when all of them are closed.

Basically what I would like to have is a dummy contact sensor that show as open when one or more window is open.

**Comment by e-mcgee** [comment]
Thanks for the positive feedback!
It is s limitation of the Home App automation unfortunately.
I have done some interesting automations using Node Red, e.g. using multiple Sonoff switches to control the same light on both sides of a hallway.
I suggest you consider using the MQTT function of the plugin to report the sensor status combined with Node Red to do your complex automation.  You will need an MQTT broker though (Mosquitto) and install Node Red. You can then simply publish the contact sensors states to the broker. You can then do what you are trying to achieve using Node Red to monitor all the sensors' statuses  to build the multiple inputs to trigger the off state and then only trigger the on state when all the contact sensors are closed.
